### PR TITLE
[#5333] Add column title row to table column output in Gravitino CLI

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Add column title row to table column output.

### Why are the changes needed?

So it's easier for a user to understand the output.

Fix: #5333

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

Compiled and tested locally.

### CODE REVIEW

File: clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListColumns.java [hunk 1]
```diff
@@ -67,8 +67,12 @@ public void handle() {
...
       String dataType = columns[i].dataType().simpleString();
       String comment = columns[i].comment();
-      String nullable = columns[i].nullable() ? "null" : "";
-      String autoIncrement = columns[i].autoIncrement() ? "auto" : "";
+      String nullable = columns[i].nullable() ? "true" : "false";
+      String autoIncrement = columns[i].autoIncrement() ? "true" : "false";
+
+      if (i == 0) {
+        all.append("name,datatype,comment,nullable,auto" + System.lineSeparator());
+      }
```
Comment (jerryshao) [inline clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListColumns.java:75]: 1) I think you can move this line above, out of "for loop", so you don't have to check if `i == 0`;
2) It is better to change "auto" to "auto_increment" for better understanding.

Comment (justinmclean) [reply clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListColumns.java:75]: 1. Not really, as you don't want to output the header if there is no data.
2. Sure I do that.

Comment (shaofengshi) [reply clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListColumns.java:75]: Justin, the column header name is still "auto", not "auto_increment", can you confirm it?

Comment (justinmclean) [reply clients/cli/src/main/java/org/apache/gravitino/cli/commands/ListColumns.java]: "I'll do that" means I do that in the future, not that it has been done. It has been done now.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**